vortexasdk/client.py

```diff
@@ -154,15 +154,18 @@ def _handle_response(response: Response, payload: Dict = None) -> Dict:
         logger.error(response.reason)
         logger.error(response.status_code)
         logger.error(response)
+        error = "[" + str(response.status_code) + " " + response.reason + "] "
```

Comment:
`f"{variable name} with surrounding sentence written here"` would be a nice way to do this